package.json

```diff
@@ -271,6 +271,7 @@
     "shasum": "1.0.2",
     "source-map-support": "0.5.19",
     "spectron-fake-dialog": "0.0.1",
+    "tail": "2.2.4",
```

Comment:
@danielmain what do you think of introducing another npm module for tailing logs? I know you raised concerns about adding too many packages to the project multiple times. This one seems to serve a valid purpose, on the other hand it could be reimplemented with probably 20-30 lines of code, so I thought it might be worth discussing this.

Reply:
I checked this dependency and we might want to move that single file in our repo maybe? https://github.com/lucagrulla/node-tail/blob/master/src/tail.js what do you think?

Reply:
Hm, I was actually thinking of implementing it from scratch using `fs.watchFile` instead of bringing another module which is more flexible but also provides functionality we don't use. Copying the file as-is feels less safe than using a npm module, where community/contributors can report and fix potential security vulnerabilities.

Reply:
@marcin-mazurek I removed the dependency and copied over only the necessary code for watching files with fixed configuration.

Reply:
Are we sure copying the implementation into our source code is less costfull than bringing another dependency? This is an additional implementation that we will need to understand and maintain. One of the purposes of public npm modules is that you trust the owner of particular module that it works correctly - the responsibility of testing and maintaining that particular implementation is not on your shoulders.

If we want to keep that implementation then we need to have a unit tests for it.